Tests/Unit/Magento/CartTest.php

```diff
@@ -0,0 +1,329 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\MagentoConnectorBundle\Tests\Unit\Magento;
+
+use ONGR\ElasticsearchBundle\ORM\Manager;
+use ONGR\ElasticsearchBundle\ORM\Repository;
+use ONGR\MagentoConnectorBundle\Document\ProductDocument;
+use ONGR\MagentoConnectorBundle\Magento\Cart;
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+/**
+ * Tests Magento Cart.
+ */
+class CartTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * Tests setters and getters.
+     */
+    public function testSettersGetters()
```

Comment:
I believe we have a test class for getters and setters. Why wasn't it used here?